dev-client/src/screens/BottomTabsScreen.tsx

```diff
@@ -38,19 +37,20 @@ import {useDispatch} from 'terraso-mobile-client/store';
 export const BottomTabsScreen = memo(() => {
   const dispatch = useDispatch();
   const {keyboardStatus} = useKeyboardStatus();
-  const [potentiallyOutdatedLocationPermission, _, requestLocationPermission] =
-    useForegroundPermissions();
+  const [locationPermissions, requestLocationPermissions, _] =
+    useUpdatedForegroundPermissions();
```

Comment:
question: am i right in understanding that the expo hooks return `[permission, get, request]` and the new hook returns `[permission, request, get]`? should we keep the same order?

Reply:
Nope! The expo hooks return [permission, request, get] 
https://docs.expo.dev/versions/latest/sdk/location/#useforegroundpermissionsoptions 

Ha, this got me too while I was working on this -- I'd been under the same impression and spent a while being very confused because I'd been calling request() when I thought I was calling get() and vice versa. 😅 

This is primarily why I'm like "hmm should I just bite the types bullet and return an object with named types instead of a list?"

Reply:
Ok I have done it

Reply:
i see, so this is actually fixing a bug? since the `request` swaps places in this diff? & i don't feel a way about it if you don't want to bother with the object with named types so up to you!

Reply:
Ah, yes, it was fixing a bug! Sorry I think my eyes had glazed over the actual code in question here, oops

And I have just finished bothering with the named things!